CHANGELOG

```diff
@@ -1,4 +1,30 @@
 # Changelog
+## [0.2.5] - 2020-02-19
+### Breaking changes
+ - Deprecate positional arguments in the following methods:
+   - "libnmstate.apply()"
+   - "libnmstate.commit()"
+   - "libnmstate.rollback()"
+   - "libmstate.show()"
+
+### New Features
+ - Report missing NM plugin during validation.
+ - Support OVS Link Aggregation (LAG).
```

Comment:
IMHO it should say "Bonding" instead of "LAG".

Reply:
Done, thank you!